cockroachcloud/authorization.md

```diff
@@ -5,11 +5,11 @@ toc: true
 docs_area: manage
 ---
 
-{{ site.data.products.db }} has a two-level authorization model: users have permissions within organizations, and on specific database clusters within organizations.
+{{ site.data.products.db }}'s authorization model has two levels: the {{ site.data.products.db }} organization and clusters within that organization.
 
-In CockroachDB Cloud, an *Organization* corresponds to a an authorization hierarchy rooted in a billing account. All database clusters in {{ site.data.products.db }} belong within an organization. The user of the billing account can create or invite other CockroachDB Cloud users to the organization. 
+A CockroachDB Cloud *organization* corresponds to a an authorization hierarchy rooted in a billing account. Each {{ site.data.products.db }} cluster is created within an organization. The user of the billing account can create or invite other users to the organization, and can grant roles to those users, such as **Cluster Admin** or **Cluster Creator**.. 
```

Comment:
Maybe use the Cloud variable.

Maybe something like "The user who created the organization is the owner of the billing account, and can create and invite other users...." WDYT?

Reply:
this stuff is changing in the FGAC PR, so not trying to over-optimize here. I agree your line is clearer for what it's worth.